### PR TITLE
[feature] two-factor auth

### PR DESCRIPTION
**Pull request by snopoke**
Dev: https://github.com/czue/wedding-plan/pull/68

### CODE REVIEW

File: 2fa.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+[1]: https://django-allauth-2fa.readthedocs.io/en/latest/
+[2]: https://django-otp-official.readthedocs.io/en/latest/
+[3]: https://www.intenct.nl/projects/django-allauth/
+
+When using Two-Factor Auth with Pegasus, a new section is added to the user profile
+for enabling & configuring the OTP devices for the user.
+
+If a user has a Two-Factor device configured then they will be prompted for a token after logging in.
+
+If you wish to force user's to enable Two-Factor Auth on their account you can follow the example
```
Comment (czue) [inline 2fa.md:15]:
```suggestion
If you wish to force users to enable Two-Factor Auth on their account you can follow the example
```

File: 2fa.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Two-Factor Authentication
+
+Two-Factor auth is configured using the [django-allauth-2fa][1] library which adapts
```
Comment (czue) [inline 2fa.md:3]:
```suggestion
Two-Factor authentication (2FA) is configured using the [django-allauth-2fa][1] library which adapts
```

File: 2fa.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+[django-otp][2] to make it work with [django-allauth][3].
+
+[1]: https://django-allauth-2fa.readthedocs.io/en/latest/
+[2]: https://django-otp-official.readthedocs.io/en/latest/
+[3]: https://www.intenct.nl/projects/django-allauth/
+
+When using Two-Factor Auth with Pegasus, a new section is added to the user profile
+for enabling & configuring the OTP devices for the user.
```
Comment (czue) [inline 2fa.md:11]:
```suggestion
for enabling & configuring the OTP (one-time password) devices for the user.
```

**Review by czue — APPROVED** [review]

**Comment by czue** [comment]
I think this should be a PR into develop, yeah? (I can change the default branch on the repo)

**Comment by snopoke** [comment]
> I think this should be a PR into develop, yeah? (I can change the default branch on the repo)

yes, true. I see the develop branch was deleted though so we'll need to re-create it.

**Comment by snopoke** [comment]
> > I think this should be a PR into develop, yeah? (I can change the default branch on the repo)
> 
> yes, true. I see the develop branch was deleted though so we'll need to re-create it.

I've re-created the develop branch off `main`

**Pull request merged**